eahub/base/static/scripts/report_buttons.js

```diff
@@ -10,6 +10,12 @@ let report_group_inactive = {
   togglers: document.getElementsByClassName('report_group_inactive_toggler')
 }
 
+let report_profile_abuse = {
```

Comment:
What does this code do?

Reply:
Right, this is no longer necessary - left from a previous implementation I abandoned. Will delete, thanks